test/runtests.jl

```diff
@@ -309,3 +278,5 @@ datadir = joinpath(@__DIR__,"data")
     @test nitems(table) == 36
   end
 end
+
+rm(savedir, recursive = true)
```

Comment:
I think we don't need to remove the temporary directory manually, the mktempdir function will do it upon exit.